src/tests/ftest/util/ior_utils.py

```diff
@@ -89,7 +86,7 @@ def __init__(self, namespace="/run/ior/*"):
         Args:
             namespace (str, optional): path to yaml parameters. Defaults to "/run/ior/*".
         """
-        super().__init__(namespace, "ior")
+        super().__init__(namespace, "ior", timeout=60)
```

Comment:
Should this be `timeout=30` and then we add the following to ior/crash.yaml:
```
ior:
  pattern_timeout: 60
```

Reply:
I was thinking that if ior/crash could take longer than 30 seconds, then other tests could take longer than 30 seconds. There isn't anything special about the way ior/crash uses IOR

Reply:
Reasonable logic :)

Reply:
FWIW, I think ior/crash is the only test using this function for IOR. Which is why we don't see the issue elsewhere